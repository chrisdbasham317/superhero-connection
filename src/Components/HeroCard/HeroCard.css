.section--hero-card {
  align-items: center;
  background: repeating-radial-gradient(black, transparent  90px);
  border: 2px solid black;
  border-radius: 5px;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  margin: 1vh 1vh;
  max-width: 160px;
}

.div--view-more-btn {
  border-radius: 20px;
  background-color: black;
  height: 20px;
  margin-bottom: 1vh;
  text-align: center;
  width: 70%;
}

.link--view-more-btn {
  text-decoration: none;
  color: white;
}