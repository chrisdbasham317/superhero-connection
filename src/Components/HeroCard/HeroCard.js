import React from 'react';


export const HeroCard = ({ }) => {
  return (
    <h1>HeroCard</h1>
  )
}

export default HeroCard;