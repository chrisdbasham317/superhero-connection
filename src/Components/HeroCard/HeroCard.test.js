import React from 'react';
import { shallow } from 'enzyme';
import HeroCard from './HeroCard';

describe('HeroCard', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<HeroCard
      id={3}
      name={'Superman'}
      img={'thisIsAnImage'}
    />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});