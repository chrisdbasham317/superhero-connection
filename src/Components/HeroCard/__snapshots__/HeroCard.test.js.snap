// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroCard should match the snapshot 1`] = `
<section
  className="section--hero-card"
>
  <img
    alt="picture of Superman"
    className="img--card-pic"
    src="thisIsAnImage"
  />
  <h2>
    Superman
  </h2>
  <div
    className="div--view-more-btn"
  >
    <Link
      className="link--view-more-btn"
      to={
        Object {
          "param": 3,
          "pathname": "/hero/3}",
        }
      }
    >
      More Info
    </Link>
  </div>
</section>
`;
